examples/tpm2-seal-unseal/main.go

```diff
@@ -0,0 +1,281 @@
+// Copyright (c) 2018, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command tpm2-seal-unseal illustrates utilizing the TPM2 API to seal and unseal data.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+var (
+	// Default EK template defined in:
+	// https://trustedcomputinggroup.org/wp-content/uploads/Credential_Profile_EK_V2.0_R14_published.pdf
+	// Shared SRK template based off of EK template and specified in:
+	// https://trustedcomputinggroup.org/wp-content/uploads/TCG-TPM-v2.0-Provisioning-Guidance-Published-v1r1.pdf
+	srkTemplate = tpm2.Public{
+		Type:       tpm2.AlgRSA,
+		NameAlg:    tpm2.AlgSHA256,
+		Attributes: tpm2.FlagFixedTPM | tpm2.FlagFixedParent | tpm2.FlagSensitiveDataOrigin | tpm2.FlagUserWithAuth | tpm2.FlagRestricted | tpm2.FlagDecrypt | tpm2.FlagNoDA,
+		AuthPolicy: nil,
+		RSAParameters: &tpm2.RSAParams{
+			Symmetric: &tpm2.SymScheme{
+				Alg:     tpm2.AlgAES,
+				KeyBits: 128,
+				Mode:    tpm2.AlgCFB,
+			},
+			KeyBits:    2048,
+			Exponent:   0,
+			ModulusRaw: make([]byte, 256),
+		},
+	}
+)
+
+func main() {
+	tpmPathFlag := flag.String("tpm-path", "/dev/tpm0", "Path to the TPM device (character device or a Unix socket).")
```

Comment:
Remove `Flag` suffix from these vars and move them up to package-level var block.
It's a much more common location for flags

Reply:
I was trying to hide these from the other methods that accept these via their arguments. WDYT?

Reply:
If their arguments have the same names, those arguments will shadow the global flag var names. So you can't accidentally use them instead.

Reply:
In general, reducing the scope of declarations and avoiding overlapping variable names in sub scopes leads to improved readability. However, I'll change this.